resources/prometheus/federation-config.yaml

```diff
@@ -1,4 +1,172 @@
 match[]:
-  # Federate all platform metrics except ACS metrics. We want all base metrics for the Kubernetes mixin.
-  - '{__name__=~".*", job!="central"}'
-  - '{__name__=~".*", job="central", endpoint!="monitoring-tls"}'
```

Comment:
FYI we need to exclude `monitoring-tls` to avoid double scraping metrics from OpenShift monitoring Prometheus (which also scrapes Central).

Reply:
This is a really important point! Thanks for noticing it.

I have added a solution to use a filter like this: `{job!~"central|scanner"}`. If we face, in the future, jobs that have these names and metrics required for Alerts or Dashboards, we need to adjust federated metrics.

We could go with a solution to duplicate each metric with both filters:
```
abc{__name__=~".*", job!="central"}
abc{__name__=~".*", job="central", endpoint!="monitoring-tls"}
```
but I think it is unnecessary.

If we would ignore the filter, we could face duplicates because some metrics like `go_memstats_alloc_bytes` - are also returned by the central `/metrics` endpoint.

We could also consider removing job (scrape) configuration for centrals in `rhacs-observability` Prometheus and only rely on federated metrics. But that should be a separate task.